package.json

```diff
@@ -25,12 +25,12 @@
     "coreUpgrade": "mvn test -Dcucumber.filter.tags='@selenium and @initialSetup and @upgrade'",
     "ocl": "./node_modules/.bin/cypress-tags run -eTAGS='@cypress and @ocl'",
     "microfrontend": "cypress-tags run -eTAGS='@cypress and @microfrontend'",
-    "refapp3Login": "cypress run --spec src/test/resources/features/refapp-3.x/01-login/login.feature",
```

Comment:
Any reason for changing the feature folders here

Reply:
The refapp-3.x features have been removed from features folder  to avoid [cucumberOptions()](https://github.com/openmrs/openmrs-contrib-qaframework/blob/dc522ce353fe9c9f53ae01acdf72420aa35b7187/src/test/java/org/openmrs/contrib/qaframework/RunTest.java#L20) not to track them as unimplemented. And thus making a chance in the scripts that locaates the respective feature files.

Reply:
Was it failing initially ?